expression/integration_test.go

```diff
@@ -2255,6 +2252,36 @@ func (s *testIntegrationSuite2) TestBuiltin(c *C) {
 	result.Check(testkit.Rows("13835058000000000000 13835058000000000000", "13835058000000000000 13835058000000000000"))
 	tk.MustExec(`drop table tb5;`)
 
+	// test builtinCastRealAsIntSig
+	tk.MustExec(`create table tb5(a double, b float);`)
+	tk.MustExec(`insert into tb5 (a, b) values (184467440737095516160, 184467440737095516160);`)
+	_ = tk.MustQuery(`select * from tb5 where cast(a as unsigned int)=0;`)
```

Comment:
We can use tk.MustExec directly.

Reply:
Check this result is nessessary in test. Please use `tk.MustQuery().Check()`